src/pyasm/web/html_wdg.py

```diff
@@ -851,7 +855,24 @@ def add_update_expression(my, expr):
         } )
 
 
+    def get_timezone_value(cls, value):
```

Comment:
I guess. so HtmlElement.eval_update(update) can contain a key called format and pass in a special timestring format. I can add that.